cli/cmd/alert_comment.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 var (
-	// alertCommentCmd represents the lql validate command
+	// alertCommentCmd represents the alert comment command
 	alertCommentCmd = &cobra.Command{
 		Use:   "comment <alert_id>",
-		Short: "Add a comment.",
+		Short: "Add a comment",
 		Long: `Post a user comment on an alert's timeline .
```

Comment:
should we validate that the alert ID exists before trying to add a comment? that is, before typing the comment via the EDITOR.

Reply:
This requires another round trip to the API.  We've gone back and forth with what sort of pre-validation we want to do on the client vs. server.  I will defer to your judgement here.

Reply:
I think that is what we did with `lacework query update <id>` 🤔 - I would suggest us doing that
extra API call to make sure the customer doesn't start writing the comment and then, find out
that the id was wrong.

Reply:
I think this could also apply to `lacework alert close` so I will file a Jira for us to follow up on this as an enhancement.

Reply:
https://lacework.atlassian.net/browse/ALLY-1261